conans/client/generators/virtualenv.py

```diff
@@ -202,8 +219,8 @@ def _call_files(flavor, activate_tpl, deactivate_tpl, file_ext=None):
 
         os_info = OSInfo()
         if os_info.is_windows and not os_info.is_posix:
-            _call_files('cmd', cmd_activate_tpl, cmd_deactivate_tpl, 'bat')
-            _call_files('ps1', ps1_activate_tpl, ps1_deactivate_tpl)
-        _call_files("sh", sh_activate_tpl, sh_deactivate_tpl)
+            _call_files(self.CMD_FLAVOR, cmd_activate_tpl, cmd_deactivate_tpl, 'bat')
+            _call_files(self.PS1_FLAVOR, ps1_activate_tpl, ps1_deactivate_tpl)
+        _call_files(self.SH_FLAVOR, sh_activate_tpl, sh_deactivate_tpl)
```

Comment:
I would use always `VirtualEnvGenerator. XXXX` or `self.XXX`, but do not mix both of them in the same class (at least not in the same PR)